lib/hanami/cli/commands/app/generate/command.rb

```diff
@@ -25,15 +25,17 @@ class Command < App::Command
             def initialize(
               fs:,
               inflector:,
-              generator_class: nil,
               **opts
             )
-              raise "Provide a generator class (that takes fs and inflector)" if generator_class.nil?
-
-              super(fs: fs, inflector: inflector, **opts)
+              super
               @generator = generator_class.new(fs: fs, inflector: inflector, out: out)
             end
 
+            def generator_class
+              # Must be implemented by subclasses, with class that takes:
+              # fs:, inflector:, out:
+            end
```

Comment:
Ahhhh, you did an equivalent to what I suggested in https://github.com/hanami/cli/pull/186, so we're deinfitely on the same page here 😄